i/intel-clc/stone.yaml

```diff
@@ -1,40 +0,0 @@
-#
-# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
-#
-# SPDX-License-Identifier: MPL-2.0
-#
-name        : intel-clc
```

Comment:
There's a `removals.md` in the project root, can you add `intel-clc` and `intel-clc-dbginfo` to it with a description of why they're being deprecated?

Reply:
Added to the removals.md